cli/common/utils_test.go

```diff
@@ -114,40 +114,16 @@ func TestGetInstancesFromArgs(t *testing.T) {
 	var args []string
 	var instances []string
 
-	projectName := "myapp"
-
-	// wrong format
-	args = []string{"myapp.instance-1", "myapp.instance-2"}
```

Comment:
If it is possible and not too time-consuming, we should have tests for our new approach.

Reply:
Added unit test